addons/project/models/project.py

```diff
@@ -1635,6 +1635,7 @@ def create(self, vals_list):
         # The sudo is required for a portal user as the record creation
         # requires the read access on other models, as mail.template
         # in order to compute the field tracking
+        was_in_sudo = self.env.su
         if is_portal_user:
```

Comment:
We think this could alter the context in your use case.
```suggestion
        was_in_sudo = self.env.su
        if is_portal_user and not was_in_sudo:
```

cc: @xavierbol

Reply:
After having double checked. In your use case the context is empty, don't change anything
Your rock 🤘🏼
Thanks 😘